runtime/lib/ttnn/operations/pool/maxpool2d.cpp

```diff
@@ -7,6 +7,31 @@
 #include "tt/runtime/ttnn/operations/utils.h"
 
 namespace tt::runtime::ttnn::operations::pool {
+
+static ::ttnn::Tensor
```

Comment:
Can we also link the TTNN/Metal issue here as a reference for the potential cleanup of custom runtime op handlings? :))

Reply:
I'm afraid I don't know which issue this is. Or do you want me to make on in metal?

Reply:
Can you create one? As this is a workaround, it'll be good to have a followup on tt-metal to fix this behavior :)) Can you create one with the description and repro?

Reply:
I'm not sure I understand exactly what the issue is. I was under the impression that we'd eventually model the shard configurations in MLIR, but to use this in maxpool for now to bringup resnet. If that's the case then I'm not sure if anything is actually wrong with TTNN maxpool2d, at least in this context.